src/components/sidebar/tabs/NodeLibrarySidebarTab.vue

```diff
@@ -145,4 +164,24 @@ const handleNodeHover = async (
     nodePreviewStyle.value.left = `${targetRect.left - 400}px`
   }
 }
+
+const toggleNode = (id: string) => {
+  if (id in expandedKeys.value) {
+    delete expandedKeys.value[id]
+  } else {
+    expandedKeys.value[id] = true
+  }
+}
+
+const insertNode = (nodeDef: ComfyNodeDef) => {
```

Comment:
nodeDef is `ComfyNodeDefImpl` type. Do note that `ComfyNodeDefImpl` is not fully compatible with `ComfyNodeDef` as the `output` type changed from a list of string to a list of objects. `app.addNodeOnGraph` accepts both types BTW.